test/features/install_jobs_service.feature

```diff
@@ -43,4 +43,27 @@ Feature: Install Kogito Jobs Service
     And Scale Kogito Jobs Service to 0 pods within 2 minutes
     And Scale Kogito Jobs Service to 1 pods within 2 minutes
 
-    Then HTTP GET request on service "kogito-jobs-service" with path "jobs/1" is successful within 1 minutes
\ No newline at end of file
+    Then HTTP GET request on service "kogito-jobs-service" with path "jobs/1" is successful within 1 minutes
+
+#####
```

Comment:
leftover?

Reply:
nope, this is made to have a clear view of separation between the tests :)